### DISCUSSION TITLE
Clip.js command

### DISCUSSION THREAD

**Discussion by mihawkman** [eritislami/evobot#419] [Q&A]
The command doesnt seem to work....The bot join the voice channel and leaves instantly...Did i do something wrong? Tq

**Comment by eritislami** [comment]
Did you put a mp3 file in sounds folder?

**Reply by mihawkman** [reply]
i did

**Reply by Maks1116** [reply]
It might be some unsupported codec (if there is one). I have used this command and everything is ok.

**Reply by mozzi-berilinky** [reply]
This may not be the correct solution, but if the mp3 file name contains spaces, remove all spaces and try again.
In this case I solved the problem.
